patches/all/oclif+1.18.1.patch

```diff
@@ -196,14 +196,33 @@ index fabe5c4..e32dd76 100644
              await qq.mv([nodeBase, 'node.exe'], cache);
              qq.popd();
          }
-diff --git a/node_modules/@oclif/dev-cli/lib/util.js b/node_modules/@oclif/dev-cli/lib/util.js
-index b3d48b7..540bbe6 100644
---- a/node_modules/@oclif/dev-cli/lib/util.js
-+++ b/node_modules/@oclif/dev-cli/lib/util.js
-@@ -40,3 +40,47 @@ function sortBy(arr, fn) {
- }
- exports.sortBy = sortBy;
- exports.template = (context) => (t) => _.template(t || '')(context);
+diff --git a/node_modules/oclif/lib/upload-util.js b/node_modules/oclif/lib/upload-util.js
+index 45392cb..3c806c7 100644
+--- a/node_modules/oclif/lib/upload-util.js
++++ b/node_modules/oclif/lib/upload-util.js
+@@ -28,10 +28,10 @@ function templateShortKey(type, ext, options = { root: '.' }) {
+     const templates = {
+         baseDir: '<%- bin %>',
+         unversioned: '<%- bin %>-<%- platform %>-<%- arch %><%- ext %>',
+-        versioned: '<%- bin %>-v<%- version %>-<%- sha %>-<%- platform %>-<%- arch %><%- ext %>',
+-        manifest: '<%- bin %>-v<%- version %>-<%- sha %>-<%- platform %>-<%- arch %>-buildmanifest',
+-        macos: '<%- bin %>-v<%- version %>-<%- sha %>.pkg',
+-        win32: '<%- bin %>-v<%- version %>-<%- sha %>-<%- arch %>.exe',
++        versioned: '<%- bin %>-v<%- version %>-<%- platform %>-<%- arch %><%- ext %>',
++        manifest: '<%- bin %>-v<%- version %>-<%- platform %>-<%- arch %>-buildmanifest',
++        macos: '<%- bin %>-v<%- version %>.pkg',
++        win32: '<%- bin %>-v<%- version %>-<%- arch %>.exe',
```

Comment:
I removed the sha from the generated files since the current scripts expect it to not be there - ideally they'd handle the fact that the oclif generated files do include the sha but for now this is the minimal diff